traffic_ops/testing/api/v14/federations_test.go

```diff
@@ -93,4 +93,16 @@ func PostTestFederationsDeliveryServices(t *testing.T) {
 	if _, err = TOSession.CreateFederationDeliveryServices(fedID, []int{ds.ID}, true); err != nil {
 		t.Errorf("creating federations delivery services: %v\n", err)
 	}
+
+	// Test get created Federation Delivery Service
+	fedDSes, _, err := TOSession.GetFederationDeliveryServices(fedID)
+	if err != nil {
+		t.Errorf("cannot GET Federation DeliveryServices: %v\n", err)
```

Comment:
Should be a `t.Fatalf` - the caller can't safely assume that `fedDSes` in this case isn't `nil`.

I know that the way you wrote that this isn't exactly possible, but in general it's true. I guess I don't really have a problem with that, provided the GoDoc notes that.